scripts/docker_run

```diff
@@ -3,11 +3,22 @@
 set -o errexit
 set -o xtrace
 
-docker build --tag=oak .
+readonly USERNAME="$(id --user --name)"
+
+mkdir -p "./bazel-cache"
+mkdir -p "./cargo-cache"
+
+docker build \
+  --tag=oak \
+  --build-arg "USERNAME=$USERNAME" \
```

Comment:
(nit) I would also add the `=` after `--build-arg` for consistency with all the other key-value flags.